doc/scenario.md

```diff
@@ -438,6 +438,7 @@ Input facts (set by role):
 
 Input facts (set by config):
 
+- `cartridge_cluster_cookie` - cluster cookie for all cluster instances (is needed to check if configuration file was changed);
```

Comment:
Add the same to other tasks

Reply:
It's always there.

Reply:
No, it skipped in doc for restart_instance